src/bhost_fpga.py

```diff
@@ -22,19 +22,10 @@
 
 class FpgaBHost(FpgaHost):
```

Comment:
@brickZA  - why not inherit from/extend FpgaXHost? A b-engine never exists on its own, but is always an addition to an x-engine board.